docs/manage/fleet/provision/setup.md

```diff
@@ -333,15 +335,15 @@ sudo ./preinstall.sh
 ```
 
 Follow the instructions.
-If you created a <FILE>viam-provisioning.json</FILE>, specify its location as an environment variable or when prompted.
+If you created a <FILE>viam-defaults.json</FILE>, specify its location as an environment variable or when prompted.
 
 You can set optional arguments by defining the following environment variables:
 
 <!-- prettier-ignore -->
 | Argument | Description |
 | -------- | ----------- |
 | `VIAM_JSON_PATH` | The path to the machine credentials <FILE>viam.json</FILE> file to be copied to the machine. The script will also prompt you for this file if not provided. |
-| `PROVISIONING_PATH` | The path to the <FILE>viam-provisioning.json</FILE> file. The script will also prompt you for this file if not provided. |
+| `DEFAULTS_PATH ` | The path to the <FILE>viam-defaults.json</FILE> file. The script will also prompt you for this file if not provided. |
```

Comment:
As I was worried about before, the version of the preinstall script with this change made wasn't yet uploaded. I've uploaded it now, so we should be good.